spyder/plugins/ipythonconsole.py

```diff
@@ -1390,6 +1390,18 @@ def process_finished(self, client):
         if self.variableexplorer is not None:
             self.variableexplorer.remove_shellwidget(id(client.shellwidget))
 
+    def connect_external_kernel_to_varexp(self, shellwidget):

```

Comment:
I forgot about connecting to our Help, so let's rename this to `connect_to_external_kernel`